recipes/py-hunspell/meta.yaml

```diff
@@ -0,0 +1,46 @@
+{% set name = "py-hunspell" %}
+{% set pypi_name = "hunspell" %}
+{% set version = "0.5.5" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ pypi_name[0] }}/{{ pypi_name }}/hunspell-{{ version }}.tar.gz
+  sha256: 0f830b68bd8c392f4d5b4e21c38e28809e14d64ec67bde48272c920b63686f53
+
+build:
+  skip: True  # [not linux64]
+  number: 0
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+  host:
+    - python
+    - pip
+    - hunspell
+  run:
+    - python
+    - hunspell
+    - hunspell-en
```

Comment:
Is `hunspell-en` required at runtime if the user isn't interested in English spellchecking?  Perhaps you could have a post-link script that looks at locale information and suggests that the user installs the correct `hunspell-XX` package for their locale?

Reply:
Oh, looks like `hunspell` already has one https://github.com/conda-forge/hunspell-feedstock/blob/main/recipe/post-link.sh

I think you should move `hunspell-en` into `test.requires` so that your test will work but you shouldn't require everyone using `py-hunspell` to install `hunspell-en`

Reply:
Done! Thank you for the feedback!